app/controllers/publishers/vacancies/documents_controller.rb

```diff
@@ -1,6 +1,8 @@
 require "google/apis/drive_v3"
 
 class Publishers::Vacancies::DocumentsController < Publishers::Vacancies::BaseController
+  include Publishers::Wizardable
```

Comment:
I don't think we need this as the `Publishers::Vacancies::BaseController` includes `Publishers::Wizardable`